src/data.toml

```diff
@@ -16,5 +16,5 @@ ios_app_icons = "When sideloading with a paid certificate the application will g
 addon_404 = "Please do not visit the URL directly since it won't display anything useful. Instead, copy the link and use it to install the add-on."
 cloud_sync_not_working = "You might be using an outdated version of Cloud Sync.\n\n- Ensure you have the [latest version](<https://bunny.nexpid.xyz/cloud-sync/>) installed.\n- Reinstall the [antied plugin](<https://angelix1.github.io/MP/angel/antied/>) if you use it.\n\nPerform a local backup to avoid losing data permanently."
 android_15_error = "Refer to https://discord.com/channels/1196075698301968455/1205101050739167272/1264529990997905442"
-fake_profile_theme_and_effects_guide = "## How to Use the Fake Profile Themes and Effects Plugin\n\n1. Select the primary color.\n2. Select the accent color.\n3. Select an effect.\n4. Copy FPTE.\n5. Paste what you just copied into the \"About Me\" section, starting on a new line at the end.\n\n**Note:** You *might* experience issues when using the Gboard clipboard to paste the FPTE. Instead, use the hold-to-paste method.\n\n![Example](https://imgur.com/paPTG3J)"
+fake_profile_theme_and_effects_guide = "## How to Use the Fake Profile Themes and Effects Plugin\n\n1. Select the primary color.\n2. Select the accent color.\n3. Select an effect.\n4. Copy FPTE.\n5. Paste what you just copied into the \"About Me\" section, starting on a new line at the end.\n\n**Note:** You *might* experience issues when using the Gboard clipboard to paste the FPTE. Instead, use the hold-to-paste method.\n\n![Example](https://imgur.com/a/U7irAXG)"
```

Comment:
what's the point of the exclamation mark, and the masked URL, those will do nothing useful, they'll just prevent discord from hiding the url.

Reply:
Removed ! Mark
Don't know who set it up like that.